presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/SingleMarkDistinctToGroupBy.java

```diff
@@ -82,6 +82,9 @@ public Pattern getPattern()
             return Optional.empty();
         }
 
+        if (!child.getOrderBySymbols().isEmpty()) {
```

Comment:
They should be used in `AggregationNode` on top of this `MDN`, so because of that they should not be used.